course_1_1/Program.cs

```diff
@@ -10,19 +10,86 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Please enter a radius");
-            string rstr = Console.ReadLine();
-            int.TryParse(rstr, out int r);
-            Console.WriteLine($"The square of the round {Program.PerformCalculation(r)}");
-            
-        }
+            ////Задание 1
+
+            //int xl;
+            //xl = 2;
+            //Console.WriteLine(xl);
+
+            ////Задание 2
+
+            //string str1;
+            //Console.WriteLine("Ваше имя?");
+            //str1 = Console.ReadLine();
+            //string str2="Привет " + str1;
+            //Console.WriteLine(str2);
+
+            ////Задание 3
+
+            //var v1 = 'v';
+            //v1 = 'r';
+            //Console.WriteLine(v1);
+
+            ////Задание 4
+
+            //Console.WriteLine("Введите длину стороны квадрата");
+            //string s = Console.ReadLine();
+            //int x = Convert.ToInt32(s);
+            //int p = 4 * x;
+            //Console.WriteLine($"Периметр квадрата {p}");
+
+            ////Задание 5
+
+            //double r1 = 8;
+            //double r2 = 6;
+            //double s1, s2,s3;
```

Comment:
Please don't use such variable declaration.
Type for each variable should be explicitly defined and placed on a separate line.